.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+  - "2.7"
+
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+  - pip install -r requirements_dev.txt
+
+before_script:
+  - sudo touch /var/log/fiware-skuld
+  - sudo chown travis /var/log/fiware-skuld
+
+# command to run tests
+script:
+  nosetests --with-coverage --cover-package=./ --cover-xml --with-xunit --exe
```

Comment:
Do we need generate XML outputs for coverage and test results (`--cover-xml` and `--with-xunit`) ?